### PR TITLE
add method to revoke token

### PR DESCRIPTION
**Pull request by hardyrarso**
Add a method to send a request to the `/openid-connect/revoke` endpoint

Let me know if I need to change anything! Thank you for maintaining gocloak.

### CODE REVIEW

**Comment by Nerzal** [comment]
Hi, if you merge the main branch the tests should be able to run now :)

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/Nerzal/gocloak/v12
+module github.com/hardyrarso/gocloak/v12
```
Comment (Nerzal) [inline go.mod:1]: i guess that is here by mistake :D

**Comment by Nerzal** [comment]
I have resolved the issue in go.mod by myself. 
Thank you for the contribution! :)

**Comment by hardyrarso** [comment]
Hello

Apologies, sorry for not responding to the comms

I opened this PR because my team needed the functionality at the time but priorities got shifted and I did not get back to you about this.

I'm glad this was able to be useful. 

Thank you! And sorry again for going AWOL

**Pull request merged**